python/google/protobuf/json_format.py

```diff
@@ -626,28 +626,28 @@ def _ConvertFieldValuePair(self, js, message, path):
                                          '{0}.{1}'.format(path, name)))
       except ParseError as e:
         if field and field.containing_oneof is None:
-          raise ParseError('Failed to parse {0} field: {1}.'.format(name, e))
+          raise ParseError('Failed to parse {0} field: {1}.'.format(name, e)) from e
```

Comment:
This line is more than 80 columns side; please shrink it.  I suggest a line-break after the comma:
```
+          raise ParseError('Failed to parse {0} field: {1}.'.format(name,
+                                                                    e)) from e
```

Same for lines 633 and 635